lstchain/data/onsite_camera_calibration_param.json

```diff
@@ -1,49 +1,65 @@
 {
    "version": 1,
-   
    "CalibrationHDF5Writer": {
      "one_event": true,
+     "events_to_skip": 1000,
      "calibration_product": "LSTCalibrationCalculator",
-     "r0calibrator_product": "LSTR0Corrections",
-     "log_level":"DEBUG"
-   },
+     "log_level":"DEBUG",   
+     "log_config": {
+       "loggers": {
+        
+       "ctapipe.CalibrationHDF5Writer.LSTEventSource":{
+        "propagate": false,
+        "level": "INFO",
+	"handlers": [
+	   "console"
+	 ]  
+       },
+       "ctapipe.io.hdf5tableio.HDF5TableWriter":{
+        "level": "INFO",
+	"handlers": [
+	   "console"
+	 ]  
+       }              
+     } 
+     }    
+    }, 
+   "LSTEventSource": {  
+    "allowed_tels": [1],
+    "LSTR0Corrections": {
+      "select_gain": false
```

Comment:
why is the gain selection set to `false` by default?

Reply:
This is the calibration config, you need to calculate pedestals / ff for both gain channels